packages/block-library/src/heading/heading-level-icon.js

```diff
@@ -17,7 +18,7 @@ export default function HeadingLevelIcon( { level } ) {
 	}
 
 	return (
-		<SVG width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+		<SVG width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" active={ active } >
```

Comment:
I'm not sure I undestand why the "SVG" component should have an "active" prop. it would make sense for a "button" to be active but not the SVG component in my opinion.

Reply:
Hey @youknowriad @gziolo as I remember gutenberg is using some css magic which allows you to define heading text color in higher-level CSS which will be passed to SVG but native doesn't support that magic. Probably we can find some better solution? But this issue is blocking our release process so we can do it after the release? Thanks.

Reply:
Yes, with CSS you can use the class names applied to parents to change how it looks. With RN it isn't possible, that's why you need to pass it down as prop.